### ISSUE TITLE
Ability to make ContextMenu follow trigger when scrolling

### ISSUE THREAD

**Issue by leadream** [radix-ui/primitives#1664]
### Overview

![image](https://user-images.githubusercontent.com/12868055/190300406-29319991-48ff-45ce-b87d-b217df3b0d17.png)

### Examples in other libraries

https://ant.design/components/dropdown-cn/#components-dropdown-demo-context-menu

**Comment by benoitgrelard** [comment]
Hi @leadream,

Are you referring to when using the context menu in "non-modal" mode?
If so I guess as scrolling is enabled, it would be fair to expect it to follow the point it was triggered at.
I'll file this as an enhancement.

**Comment by leadream** [comment]
@benoitgrelard yeah, I tested dropdown in non-modal mode and it will follow the trigger element when scrolling, but context menu doesn't.
